app/_hub/kong-inc/opentelemetry/_index.md

```diff
@@ -98,6 +98,15 @@ params:
       datatype: number
       description: |
         The timeout, in milliseconds, for reading the response from the OTLP server.
+    - name: header_type
+      required: true
```

Comment:
Is this accurate? I believe there’s a default value based on the other PR

Reply:
I believe so, below this line it specifies the default as `default: preserve`. However, in the schema of the plugin it does note that it's required, so wanted that to match here (it's similar to the Zipkin implementation too, which has these similar values).

However, if it needs to be updated let me know!